src/server/services/buurt/datasets.ts

```diff
@@ -487,6 +505,40 @@ export function transformMeldingenBuurtResponse(
   return transformDsoApiListResponse(datasetId, config, { features });
 }
 
+export async function fetchLaadpalen(requestConfig: DataRequestConfig) {
+  const urls = [
+    requestConfig.url,
+    'https://map.data.amsterdam.nl/maps/oplaadpunten?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAMES=ms:snel_beschikbaar&OUTPUTFORMAT=geojson&SRSNAME=urn:ogc:def:crs:EPSG::4326',
+  ];
+  const requests = urls?.map((url) => {
+    return axiosRequest.request<DatasetFeatures>({
+      ...requestConfig,
+      url,
+    });
+  });
+
+  let responses: any;
+
+  try {
+    responses = await Promise.all(requests);
+  } catch (error) {
+    return error;
+  }
+
+  responses = responses.filter((res: any) => res.data);
+
+  // If transformDetail is called return the first response for the detail view
+  if (responses[0].data._embedded) return responses[0];
+
+  // Combine all responses into one for the list view
+  if (responses.length > 1) {
+    responses[0].data = responses.slice(1).reduce((acc: any, response: any) => {
+      return acc.concat(response.data);
+    }, responses[0].data);
+  }
+  return responses[0];
```

Comment:
Je kent de type hier dus any is niet nodig. Je hoeft de Promise.all ook niet te wrappen in try/catch, dat gebeurt al in source-api-request. 

En ipv reduce, kun je niet beter iets doen als:

```
const responses = await Promise.all(requests);
responses[0].data = responses.map((response)=>response.data).flat();
return responses[0];
```